Containerfile

```diff
@@ -4,10 +4,9 @@ LABEL maintainer="LSIT Systems <lsitops@ucsb.edu>"
 
 USER root
 
-RUN R -e "install.packages(c('astsa'), repos = 'https://cloud.r-project.org/', Ncpus = parallel::detectCores())"
 
-RUN conda install -yc conda-forge orjson pyarrow 
+RUN conda install -yc conda-forge orjson pyarrow gluonts pytorch-forecasting r-astsa r-bsts
```

Comment:
I'm curious if this includes the extensions we were specifying with pip before or if that still matters?

Reply:
Frrom what I remember, there wasn't a specific request for them. Typically conda tends to include more than what's required and I might have just included them all originally to shotgun any potential requests because the docs mention them. Conda doesn't seem to have any extensions or variants for GluonTS like we've seen with pytorch or other packages.

Reply:
If for some reason we need the extensions hopefully pip will be able to install (just them) more easily now that the base package is in there via conda.  I'll mention it to the instructor to have them check.